src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -81,6 +81,12 @@ public function getConfigTreeBuilder()
                 ->scalarNode('ide')->defaultNull()->end()
                 ->booleanNode('test')->end()
                 ->scalarNode('default_locale')->defaultValue('en')->end()
+                ->arrayNode('available_locales')
```

Comment:
there is `enabled_locales` already, can't we reuse it?

Reply:
it's in the translator section.
what if the translator is not available?
Is it still relevant to reuse it?

Reply:
My fear is that it will be confusing for users to have to configure this twice.

Reply:
I suggest to make the translator option defaulting to the value of this new parameter if not set explicitly (and to not set it explicitly in the recipe). It’s already what we do for the default local IIRC.

Reply:
or we could move the setting out of the translation subtree.
again, having twice the same setting is calling for troubles to me.